Library/Formula/spidermonkey.rb

```diff
@@ -1,18 +1,19 @@
 require 'formula'
 
 class Spidermonkey < Formula
-  homepage 'https://developer.mozilla.org/en/SpiderMonkey'
-  url 'http://ftp.mozilla.org/pub/mozilla.org/js/js185-1.0.0.tar.gz'
-  version '1.8.5'
-  sha1 '52a01449c48d7a117b35f213d3e4263578d846d6'
-
-  head 'https://hg.mozilla.org/tracemonkey/archive/tip.tar.gz'
-
-  bottle do
-    revision 1
-    sha1 "6c6fd6d40d41764a086a6fb134176253deb1a51b" => :yosemite
-    sha1 "5d19010b10a5f1827511ca791debf9f2d9076e47" => :mavericks
-    sha1 "37d04b64aba47dbf65f197aec94da9acf5f1fd4c" => :mountain_lion
+  homepage 'https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey'
+  url 'https://ftp.mozilla.org/pub/mozilla.org/js/mozjs-24.2.0.tar.bz2'
+  version '24.2.0'
+  sha1 'ce779081cc11bd0c871c6f303fc4a0091cf4fe66'
+
+  depends_on 'yasm'
+  depends_on 'gawk'
+  depends_on 'libidl'
+  depends_on 'ccache'
+
+  head do
+    depends_on 'https://raw.github.com/Homebrew/homebrew-versions/master/autoconf213.rb'
+    url 'https://hg.mozilla.org/mozilla-central/archive/tip.tar.bz2'
```

Comment:
> And your comment would be relevant if brew was keeping the source checkout after installation, but as this is all done in a temporary directory cache, if you want to upgrade your tools, you'll actually checkout again the whole 3GB through mercurial, taking another 20 min.

It does keep it but fair enough on the `tip.tar.gz`; that size different is fair enough.